governance.md

```diff
@@ -1,12 +1,12 @@
-The Spinnaker OSS project is run by a group of appointed members who hold specific roles, managed by the technical oversight committee. The steering committee manages the governance structure of the project, revisiting and refining it as needs evolve.
+The Spinnaker OSS project is run by a group of appointed members who hold specific roles, managed by the technical oversight committee. The Technical Oversight Committee manages the governance structure of the project, revisiting and refining it as needs evolve.
```

Comment:
Capitalize here too?  Or perhaps introduce TOC here instead of lower down?
```suggestion
The Spinnaker OSS project is run by a group of appointed members who hold specific roles, managed by the Technical Oversight Committee (TOC). The TOC manages the governance structure of the project, revisiting and refining it as needs evolve.
```